ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityAdapter.java

```diff
@@ -163,8 +172,10 @@ public void onInitializationFailed(UnityAds.UnityAdsInitializationError
             }
           }
         });
-
-    UnityAds.load(placementId, unityLoadListener);
+    objectId = UUID.randomUUID().toString();
```

Comment:
Is there a possibility that this function `requestInterstitialAd` can be called multiple times?

Reply:
Yes its always possible I guess why?

Reply:
then there is a chance that Load and show could have different objcID no?

Reply:
No since its generated by load and used by show. (and both are called from the same callers in a sequence)